### ISSUE TITLE
toolbar with only custom entries not working with 0.8

### ISSUE THREAD

**Issue by sagiegurari** [varemenos/atom-toolbar-almighty#20]
Updated to latest version 0.8.0 and now my toolbar is completely missing.
I use a custom entries only toolbar.

**Comment by sagiegurari** [comment]
atom 11.1.0 beta
tool-bar-almigthy - 0.8.0
tool-bar - 1.0.1

I am using custom js file. if i check the override it all disappears, unchecked I only see your entries.

**Comment by sagiegurari** [comment]
ok so to fix the issue in utils.coffee you need to change from

``` coffee
content = fs.readFileSync(thePath, 'utf8')
```

to 

``` coffee
content = require(thePath)
```

Don't think you need a pull request for that small change.
I manually fixed my atom but would like it fixed in y our repo please.

**Comment by varemenos** [comment]
The issue with `content = require(thePath)` is that it caches the file required which was supposed to fix #13.
Thank you very much for reporting this issue, I will see if I can find an alternative way to invalidate the require cache and fix this issue

**Comment by sagiegurari** [comment]
ah now I get the change.
so a simple solution, would be to enforce json/cson structure and not js/cofffe and than just load the file and do JSON.parse for example.
that would not be cached.

**Comment by varemenos** [comment]
I've just pushed a [branch](https://github.com/varemenos/atom-toolbar-almighty/tree/bugfix/json-format) is supposed to tackle this issue.
Could you please check whether it fixes the issue at your end? I'm also testing this but it's a definite breaking change so I want to make sure it's transition is as smooth as possible

**Comment by sagiegurari** [comment]
switched from .js to .json and did the code change you like in the utils.coffee and it seems to be working good for me 👍

**Comment by varemenos** [comment]
I've just published v0.8.1 which should resolve this issue, please feel free to comment here or open a new issue if you encounter any issues. Thanks @sagiegurari for helping me out on this.

**Comment by sagiegurari** [comment]
0.8.1 works good. thanks
